tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -14,21 +14,68 @@ import (
 	"github.com/openshift/odo/tests/helper"
 )
 
-const (
-	CI_OPERATOR_HUB_PROJECT = "ci-operator-hub-project"
-)
-
 var _ = Describe("odo service command tests for OperatorHub", func() {
 
+	var project string
+
 	BeforeEach(func() {
 		SetDefaultEventuallyTimeout(10 * time.Minute)
 		SetDefaultConsistentlyDuration(30 * time.Second)
-		helper.CmdShouldPass("odo", "project", "set", CI_OPERATOR_HUB_PROJECT)
 		// TODO: remove this when OperatorHub integration is fully baked into odo
 		os.Setenv("ODO_EXPERIMENTAL", "true")
 	})
 
+	preSetup := func() {
+		project = helper.CreateRandProject()
+		helper.CmdShouldPass("odo", "project", "set", project)
+
+		// wait till oc can see the all operators installed by setup script in the namespace
+		ocArgs := []string{"get", "csv"}
+		helper.WaitForCmdOut("oc", ocArgs, 1, true, func(output string) bool {
+			return strings.Contains(output, "NAME")
+		}, true)
+
+		ocArgs = []string{"get", "csv"}
+		helper.WaitForCmdOut("oc", ocArgs, 1, true, func(output string) bool {
```

Comment:
How if you execute this `helper.waitForCmdOut` in a loop because only one arg is getting changed? And yes the loop style implementation will look programmatically smart.

Reply:
It's two things now - etcd and mongodb. IMO, it's OK to not use loop for it.

Reply:
Atm its fine but what if we expand the number of operators in that, then loop will really matters a lot. So lets do it from now to avoid the refactor/cleanup for someone else.

Reply:
Makes sense. Addressed.